src/Installing/SetupIniFile.php

```diff
@@ -42,9 +42,9 @@ public function __invoke(
                 $output->writeln('Automatic extension enabling was skipped.', OutputInterface::VERBOSITY_VERBOSE);
             }
 
-            $output->writeln('<comment>⚠️  Extension has NOT been automatically enabled.</comment>');
+            $output->writeln('<comment>⚠️  Extension is not enabled.</comment>');
```

Comment:
This messaging is not only emitted when `--skip-enable-extension` is used; it is also when we failed to automatically enable the extension. The messaging on the line above (`Automatic extension enabling was skipped.`) provides the context, although only with verbose mode.